src/main/python/rlbot/utils/structures/game_data_struct.py

```diff
@@ -121,15 +141,17 @@ def rotate_game_tick_packet_boost_omitted(game_tick_packet):
 
     # Rotate yaw 180 degrees is all that is necessary.
     ball_yaw = game_tick_packet.game_ball.rotation.yaw
-    game_tick_packet.game_ball.rotation.yaw = ball_yaw + 32768 if ball_yaw < 0 else ball_yaw - 32768
+    game_tick_packet.game_ball.rotation.yaw = ball_yaw + math.pi if ball_yaw < 0 else ball_yaw - math.pi
 
     for i in range(game_tick_packet.num_cars):
         game_tick_packet.game_cars[i].physics.location.x = -1 * game_tick_packet.game_cars[i].physics.location.x
         game_tick_packet.game_cars[i].physics.location.y = -1 * game_tick_packet.game_cars[i].physics.location.y
         game_tick_packet.game_cars[i].physics.velocity.x = -1 * game_tick_packet.game_cars[i].physics.velocity.x
         game_tick_packet.game_cars[i].physics.velocity.y = -1 * game_tick_packet.game_cars[i].physics.velocity.y
-        game_tick_packet.game_cars[i].physics.angular_velocity.x = -1 * game_tick_packet.game_cars[i].physics.angular_velocity.x
-        game_tick_packet.game_cars[i].physics.angular_velocity.y = -1 * game_tick_packet.game_cars[i].physics.angular_velocity.y
+        game_tick_packet.game_cars[i].physics.angular_velocity.x = -1 * game_tick_packet.game_cars[
```

Comment:
Formatting

Reply:
The auto-formatter did this because the line was too long. I'll break the line at a more appropriate point.

Reply:
Ultimately decided to tell the auto-formatter to shut up instead.